### ISSUE TITLE
Sample image generation during training is significantly worse than with hv_generate_video.py

### ISSUE THREAD

**Issue by Enyakk** [kohya-ss/musubi-tuner#30]
Thank you for the addition of training sample generation. This is useful to check the quality of the training and the performance is adequate.

However I have found that the generated samples produce a very blurry unfocussed image / video (and sometimes strange body anatomy) that is like seeing through a semi-transparent glass door. Meanwhile using the generated LORA safetensors with hv_generate_video.py generates very good quality results.

```
prompts.txt:
Supergirl with blonde ponytail standing in a defiant pose with her arms on her hips looking straight at the viewer. She has a slight smile on her face. the image is bright with good contrast and natural colors --w 624 --h 832 --f 25 --s 25 -d 1234
```
Maybe other users can confirm the discrepancy between training and offline samples. My suspicion is that a needed parameter is possibly missing or set up differently.

**Comment by Enyakk** [comment]
Curiously the quality is already poor for me training-sample 0 with `--sample_at_first`.

My training commandline is the following:
`accelerate launch --num_cpu_threads_per_process 1 --mixed_precision bf16 hv_train_network.py --dit "D:\ML\SD\models\Video\Hunyan Video\kohya\mp_rank_00_model_states_fp8.safetensors" --dataset_config %1 --sdpa --mixed_precision bf16 --fp8_base --optimizer_type adamw8bit --learning_rate 2.0e-3 --gradient_checkpointing --max_data_loader_n_workers 8 --persistent_data_loader_workers --network_module=networks.lora --network_dim=32 --timestep_sampling sigmoid --discrete_flow_shift 1.0 --max_train_epochs 15 --save_every_n_epochs=1 --save_last_n_steps 200 --seed 42 --output_dir %2 --output_name 2025-01-12_1230 --save_state --save_last_n_epochs_state 1 --blocks_to_swap 24 --split_attn --vae "D:/ML/SD/models/Video/Hunyan Video/hunyuan-video-t2v-720p/vae/pytorch_model.pt" --vae_chunk_size 32 --vae_spatial_tile_sample_min_size 128 --text_encoder1 D:/ML/SD/models/TE/llava-llama-3-8b-text-encoder-tokenizer --text_encoder2 D:/ML/SD/models/TE/clip-vit-large-patch14 --sample_prompts %3 --sample_every_n_steps 100 --sample_every_n_epochs 1 --sample_at_first`

Whereas %3 is then the prompt from above. This is in a Windows11 install.

**Comment by ajrss20236** [comment]
Any hint for caption an image? I trained using musubi tuner GUI and no lora works in comfyui. I converted before using.

**Comment by Enyakk** [comment]
The change was an improvement. However I am noticing a trend that the training-samples are still qualitatively worse than the offline samples, using the same prompts. For me this is more noticeable in a higher percentage of rendering defects in the training-samples: Muted colors or contorted proportions when rendering persons.

I can't offer a repro case or objective markers unfortunately and I could be more wrong. IMO more tests are needed.

**Comment by Enyakk** [comment]
I would like to close the issue for now. This can be re-opened if there is a good repro case.
